### DISCUSSION TITLE
Where can I find or recover unsaved image?

### DISCUSSION THREAD

**Discussion by P0huber** [PintaProject/Pinta#931] [Q&A]
Hello, friends

While working with Pinta 2.1.1 on Mac I had 3 images open. I created a new document, changed the canvas size and the program froze. Nothing helped to return control of the program and I forced to close the program. How can I get back the unsaved document named "unsaved image 7"?

**Comment by mcc-us** [comment]
Working on Ubuntu Linux, all my files are stubbornly sent to `/home/me/tmp` Try there...

**Comment by cameronwhite** [comment]
If the images weren't saved, then I suspect they're gone unfortunately. There isn't an auto-backup/save functionality
